src/main/java/org/traccar/protocol/NavtelecomProtocolDecoder.java

```diff
@@ -295,6 +295,81 @@ protected Object decode(
+                                    value = buf.readUnsignedShortLE();
+                                    position.set(Position.KEY_RPM, (value != 0xFFFF) ? value : null);
+                                    break;
+                                case 56:
+                                    value = buf.readByte();
+                                    position.set(Position.KEY_COOLANT_TEMP, (value != 0x80) ? value : null);
+                                    break;
+                                case 57:
+                                    position.set(Position.KEY_OBD_ODOMETER, buf.readFloatLE());
+                                    break;
+                                case 58:
+                                case 59:
+                                case 60:
+                                case 61:
+                                case 62:
+                                    value = buf.readUnsignedShortLE();
+                                    position.set(
+                                            Position.KEY_AXLE_WEIGHT + (j + 2 - 58), (value != 65535) ? value : null);
+                                    break;
+                                case 63:
+                                    value = buf.readUnsignedByte();
+                                    position.set("obdAccelPos", (value != 0xFF) ? value : null);
+                                    break;
+                                case 64:
+                                    value = buf.readUnsignedByte();
+                                    position.set("obdBrakePos", (value != 0xFF) ? value : null);
+                                    break;
+                                case 65:
+                                    value = buf.readUnsignedByte();
+                                    position.set(Position.KEY_ENGINE_LOAD, (value != 0xFF) ? value : null);
+                                    break;
+                                case 66:
+                                    value = buf.readUnsignedShortLE();
+                                    if (value == 0x7FFF) {
+                                        position.set("obdAdBlueLevel", (Boolean) null);
+                                        position.set("obdAdBlue", (Boolean) null);
+                                    } else if (BitUtil.check(value, 7)) {
+                                        position.set("obdAdBlueLevel", BitUtil.to(value, 6));
+                                        position.set("obdAdBlue", (Boolean) null);
+                                    } else {
+                                        position.set("obdAdBlueLevel", (Boolean) null);
+                                        position.set("obdAdBlue", BitUtil.to(value, 6) / 10);
+                                    }
+                                    break;
+                                case 67:
+                                    position.set("obdHours", buf.readUnsignedIntLE() / 60);
```

Comment:
This should probably be just the same parameter.

Reply:
Well, we don't have two parameters. I guess we can keep it as `obdHours` if you think the other one is more important. But the value should probably be millis.